docs/_adapters/osm/osm.md

```diff
@@ -6,13 +6,16 @@ mesh_name: Open Service Mesh
 version: v0.5.0
 port: 10009/tcp
 project_status: stable
+lab: osm-meshery-adapter
```

Comment:
```suggestion
lab: open-service-mesh-meshery-adapter
```

Reply:
How is this reference used? Is it material to the lab being displayed? The lab shows on https://61454b12d1832a84f213e937--meshery-docs-preview.netlify.app/service-meshes/adapters/osm now.

Reply:
@leecalcote this review is from before, the changes now are fine.